### ISSUE TITLE
request is big and slow

### ISSUE THREAD

**Issue by ephaee** [epha/bibliothek#18]
browserify'ing the request module builds a 1.4MB script, that slows down everything.
![screen shot 2015-06-23 at 13 25 54](https://cloud.githubusercontent.com/assets/5112286/8305081/e62935dc-19ab-11e5-855b-06c5c99d382f.png)

**Comment by meaku** [comment]
Bundling only a single module adds the overhead of the bundler runtime and polyfills. If all modules would be bundled the benefit would be greater. 

# 1 Module

- Runtime: 90%
- Modules: 10 %

# Many Modules

- Runtime: 10 %
- Modules: 90 %

Another benefit is code-reuse (schemas!), compression during bundling and optimized builds that could be cached using the application manifest. I can show you our setup if you are interested?

**Comment by ephaee** [comment]
Yes we should build all browserify scripts from one file into one file. Exporing them as globals, i.e. reguest, eventify, useragent.

No application manifest for the moment.

How exactly did you come up with the  filesize percentages? :)

**Comment by meaku** [comment]
>How exactly did you come up with the filesize percentages? :)

Just guesses.. But you're right. request is really huge. Maybe the bundle could be stripped down, but using #19 fetch would be better.
